WalletWasabi.Tests/UnitTests/Transactions/TransactionStoreTests.cs

```diff
@@ -79,5 +77,19 @@ public async Task CanDoOperationsAsync()
 			Assert.Single(txStore.GetTransactions());
 			Assert.Single(txStore.GetTransactionHashes());
 		}
+		
+		private static async Task<TransactionStore> CreateTransactionStoreAsync([CallerFilePath] string callerFilePath = "", [CallerMemberName] string callerMemberName = "")
+		{
+			// Make sure starts with clear state.
+			var dir = Path.Combine(Common.GetWorkDir(callerFilePath, callerMemberName));
+			var filePath = Path.Combine(dir, "Transactions.dat");
+			if (File.Exists(filePath))
+			{
+				File.Delete(filePath);
```

Comment:
Fun fact: `File.Delete` would suffice here. `File.Exists` is not needed actually because: 

> If the file to be deleted does not exist, no exception is thrown.
(https://docs.microsoft.com/en-us/dotnet/api/system.io.file.delete?f=&view=net-5.0)

Reply:
It fails with: `System.IO.DirectoryNotFoundException : Could not find a part of the path`